mix-and-match.yml

```diff
@@ -9,9 +9,13 @@ services:
       - CB_HOST
       - CB_USER
       - CB_PSWD
-    command: --storage.host=${CB_HOST} --storage.username=${CB_USER} --storage.password=${CB_PSWD}
-    volumes:
-      - .:/app
+    entrypoint: ["./wait-for-couchbase.sh"]
```

Comment:
Overriding entrypoint and command here as we have to pass the properties before `-jar try-cb-scala-xxx`.

Now mix and match yml works as expected.